packages/math-input/less/main.less

```diff
@@ -1,2 +1,2 @@
 @import "./overrides.less";
-@import (inline) "../node_modules/mathquill/build/mathquill.css";
+@import "mathquill/build/mathquill.css";
```

Comment:
This is the correct import syntax to resolve a CSS/Less file out of an dependant npm package.